product_docs/docs/eprs/7.0/eprs_rel_notes/19_eprs_rel_notes_7.0.1.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: "Version 7.0.1"
+---
+
+
+New features, enhancements, bug fixes, and other changes in EDB Postgres Replication Server 7.0.1 include the following:
+
+| Type        | Description |
+| ----------- |------------ |
+| Security Fix| Addresses JDBC vulnerability CVE-2022-21724 that has a CVSS score of 8.5, which is a High severity. The PostgreSQL JDBC driver version 42.3.2 that contains the security fix is bundled with Replication Server. If you have a RPM- or Debian-based Replication Server installation, the EDB JDBC driver for Advanced Server, which incorporates the community version 42.3.2, needs to be updated. For example, on RHEL 7 or CentOS 7 distributions, run: `yum update edb-jdbc`. |
+| Bug Fix     | Publication database registration fails for a SSL connection with a non-default SSL client certs/key path. [77621] |
```

Comment:
Is this simply a placeholder for actual bug fix(es) that may have been implemented in 7.0.1.  This fix was made in 6.2.17, but not sure about 7.0.1.  Would need to verify.